1.29 upgrade notes (#834)
charmed-kubernetes/kubernetes-docs: pages/k8s/1.29/release-notes.md

### Charmed Operator Framework (Ops)

We're pleased to announce the completion of the Charmed Kubernetes refactor that began
last year. Core charms have moved from the `reactive` and `pod-spec` styles to the `ops`

> framework in order to enable access to common charm libraries, gain better Juju support,

Reviewer: ```suggestion
framework. This shift aims to enable access to common charm libraries, gain better Juju support,
```
